src/Features/Core/Portable/ExtractMethod/MethodExtractor.TypeParameterCollector.cs

```diff
@@ -29,6 +29,15 @@ public override void VisitDynamicType(IDynamicTypeSymbol dynamicTypeSymbol)
             {
             }
 
+            public override void VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
+            {
+                symbol.Signature.ReturnType.Accept(this);
+                foreach (var param in symbol.Signature.Parameters)
+                {
+                    param.Type.Accept(this);
+                }
```

Comment:
do we have a test of a feature that goes through thisx codepath?

Reply:
Added a method extraction test that involves a type parameter in a function pointer.